docs/faq.rst

```diff
@@ -514,8 +514,8 @@ The Borg config directory has content that you should take care of:
   recovered.
 
 ``keys`` subdirectory
-  In this directory all your repository keyfiles are stored. This does **not** apply if you
-  use the repokey mode because your key is stored alongside the the repository. You MUST make sure to have an
+  In this directory all your repository keyfiles are stored. This does **not** apply to repokey keys because they
+  are stored alongside the the repository rather than in ``$HOME/.config/borg/keys``. You MUST make sure to have an
```

Comment:
sorry to bother again, but noticed something:

the text talks about **repository keyfiles**, which can easily be confused that it means **repokey** specifically or all keys including repokeys (which is both wrong). so guess it better should read:

```
All your borg keyfile keys are stored in this directory.
```

The next sentence then should read (shorter and easier):
```
Please note that borg repokey keys are stored inside the repository.
```

Reply:
> inside the repository


I do not understand why you insist on saying that repokey keys are stored *inside* the repository, but you are the boss. My concern is that users might think that repokey keys are stored inside the backup even though they are merely stored in the same directory as the repository.

Reply:
well, "repository" is all inside the repository directory (e.g. "repo/") and the repokey is in "repo/config", so it is inside (the directory). one could also say that it is in the repository config, but not everybody will know that the repository config is inside the repository directory.

we could check if the docs say this as clear as that, but maybe that is another place in the docs, this place is specifically about `~/.config/borg/keys`.

Reply:
ok